docs/coffee.ts

```diff
@@ -0,0 +1,304 @@
+const coffeeSelector = document.getElementById(
+  "coffeeSelector"
+) as HTMLInputElement;
+const teaSelector = document.getElementById("teaSelector") as HTMLInputElement;
+const dessertSelector = document.getElementById(
+  "dessertSelector"
+) as HTMLInputElement;
+const menuGrid = document.getElementById("menuGrid") as HTMLDivElement;
+const navModal = document.getElementById("navModal");
+let total: number;
+let products: product[];
+let productsLength: number;
+
+//creating a type
+type product = {
```

Comment:
The convention we use is to PascalCase for types, so "type Product {..."

Reply:
Changed to PascalCase.